codeclimate/data_source_repository.go

```diff
@@ -10,11 +10,15 @@ func dataSourceRepository() *schema.Resource {
 		Read: dataSourceRepositoryRead,
 
 		Schema: map[string]*schema.Schema{
-			"repository_id": &schema.Schema{
+			"repository_slug": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
 			},
-			"test_reporter_id": {
+			"repository_id": &schema.Schema{
```

Comment:
It seems `repository_id` is no longer needed.

Reply:
You're right, I initially thought that we need to keep it to store the reference, but we have an ID from terraform, which I set. Thanks